virt/backup_restore/virt-disaster-recovery.adoc

```diff
@@ -0,0 +1,14 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="virt-disaster-recovery"]
+= Disaster recovery
+include::_attributes/common-attributes.adoc[]
+:context: virt-about-dr
+
+toc::[]
+
+The disaster recovery documentation provides information for administrators on
+how to recover from certain disaster scenarios that might occur with an
+{product-title} cluster. As an administrator, you must plan your {VirtProductName} deployment in advance in order to take advantage of disaster recovery.
+
+Metropolitan Disaster Recovery (Metro-DR)::
+Metro-DR provides two-way synchronous data replication between OpenShift Virtualization clusters installed at two separate sites. Use Metro-DR during a site disaster to fail applications to the additional site. This synchronous solution is only available to metropolitan distance data centers with a 10 millisecond latency or less. For more information, see: https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.14/html-single/configuring_openshift_data_foundation_disaster_recovery_for_openshift_workloads/index#metro-dr-solution[Metro-DR solution for OpenShift Data Foundation].
```

Comment:
[1]between OpenShift Virtualization clusters -> between managed OpenShift Virtualization clusters
[2]installed at two separate sites -> installed on the primary and secondary sites.
[3] to fail applications to the additional site - to fail over applications from the primary to the secondary site and later after restoring the disaster site to relocate the application back from the secondary to the primary site.